### PR TITLE
Resolve eslint failures

### PR DESCRIPTION
**Pull request by jrschumacher**
Depends on #37 

changes: _rev_

### Proposed Changes
- resolve linting issues

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [ ] I have verified that my changes have not introduced new lint errors
- [ ] I have updated the change log

### CODE REVIEW

File: lib/components/Layout/Header.js [hunk 1]
```diff
@@ -13,31 +13,44 @@ import styles from './styles/header.css';
...
-  } else {
-    const Logo = logoWhite;
-    return <div className={styles.brandingLogo}><Logo height={20} /></div>
   }
+  const Logo = logo;
+  return (
+    <div className={styles.brandingLogo}>
+      <Logo height={20} />
+    </div>
+  );
```
Comment (avkvirtru) [inline lib/components/Layout/Header.js:26]: nice cleanup in the above, even axing the else statement :)

File: lib/components/Layout/Header.js [hunk 2]
```diff
@@ -13,31 +13,44 @@ import styles from './styles/header.css';
...
+Branding.propTypes = {
+  company: PropTypes.string,
+  logo: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
+};
+
+Branding.defaultProps = {
+  company: 'Virtru',
+  logo: logoWhite,
+};
+
```
Comment (avkvirtru) [inline lib/components/Layout/Header.js:38]: I'd consider breaking these up into multiple files at some point.

File: lib/components/Layout/Header.js [hunk 3]
```diff
@@ -13,31 +13,44 @@ import styles from './styles/header.css';
...
 /**
  * Header Component that displays content at the top of the page.
  */
 function Header({ children, company, logo, title }) {
-  const [
-    nextChildren,
-    { Menu: MenuComponents },
-  ] = expandChildren(children, [Menu]);
+  const [, { Menu: MenuComponents }] = expandChildren(children, [Menu]);
```
Comment (avkvirtru) [inline lib/components/Layout/Header.js:43]: I actually find the previous format more readable for destructuring. Does ESLint discourage it?

File: lib/components/Modal/Modal.js [hunk 4]
```diff
@@ -16,13 +16,14 @@ export const VARIANT = {
...
     <div className={styles.overlay}>
+      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
       <div className={variant === VARIANT.LARGE ? styles.large : styles.small} {...props}>
         <div className={styles.header}>
           <div className={styles.title}>{title}</div>
           <div>
             {showCloseButton ? (
               <button type="button" className={styles.closeButton} onClick={onRequestClose}>
-                {'✕'}
+                ✕
```
Comment (avkvirtru) [inline lib/components/Modal/Modal.js:26]: heh, I wonder why the original was what it was…

File: lib/components/Select/components/ClearIndicator/ClearIndicator.js [hunk 5]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { components as ReactSelectComponent } from 'react-select';
-import { vds as designTokens } from '@/styles/build/js/design_tokens.js';
+import { vds as designTokens } from '@/styles/build/js/design_tokens';
```
Comment (avkvirtru) [inline lib/components/Select/components/ClearIndicator/ClearIndicator.js:3]: nice, didn't realize the `.js` was optional…

File: lefthook.yml [hunk 6]
```diff
@@ -6,14 +6,14 @@ pre-commit:
   parallel: true
   commands:
     lint-js:
-      files: git diff --name-only HEAD @{push}
+      files: git diff --name-only HEAD master
```
Comment (avkvirtru) [inline lefthook.yml:9]: Just curious: what's the practical difference here?

**Comment by avkvirtru** [comment]
Great work with all this cleanup @jrschumacher!

**Comment by jrschumacher** [comment]
@avkvirtru i think I made a mistake when I "merged". Need to rebase only I think.

**Comment by jrschumacher** [comment]
:rocket: PR was released in [`v1.2.1`](https://github.com/avkvirtru/virtuoso-design-system/releases/tag/v1.2.1) :rocket:

**Pull request merged**